### ISSUE TITLE
Demographic Insight Stack

### ISSUE THREAD

**Issue by ThaliaBarrera** [airbytehq/quickstarts#61]
### Demographic Analysis with Airbyte

Use Airbyte to extract demographic, economic, or housing data from the US Census API. Load it into a warehouse for insightful analysis on population trends, economic indicators, or housing developments.

You can use a tool like dbt for data transformation, and an orchestrator like Airflow or Dagster if needed.

How to get started:

- Check out the [README](https://github.com/airbytehq/quickstarts/tree/main) and the [Contribution instructions](https://github.com/airbytehq/quickstarts/blob/main/CONTRIBUTING.md) in this repo.
- Check one of the [existing quickstarts](https://github.com/airbytehq/quickstarts#list-of-available-quickstarts) and observe their structure.
- (Optional, but highly recommended) Implement one of the quickstarts by following the instructions in their README. I recommend starting with [E-commerce analytics in BigQuery](https://github.com/airbytehq/quickstarts/tree/main/ecommerce_analytics_bigquery)

**Comment by WillOfHeaven** [comment]
Hey Can I Work on this ? and one more question is this also an hactoberfest repository.

**Comment by ThaliaBarrera** [comment]
Hi @WillOfHeaven! Sure, I have assigned it to you. 

Yes, these quickstarts are also rewarded as part of the Hacktoberfest :).

**Comment by WillOfHeaven** [comment]
Hey @ThaliaBarrera I am quite new to using frameworks like dagster and dbt and I would like to take some time learning them and therefore request you to unassign me so that other can have a chance while I in that time figure out these frameworks and work on them so that I can contribute.

**Comment by ThaliaBarrera** [comment]
Ok @WillOfHeaven, thanks for letting me know! I'll unassign you now.

**Comment by bishalbera** [comment]
@ThaliaBarrera If you think this [PR](https://github.com/airbytehq/quickstarts/pull/79) could be merged then I think I can solve this issue within two days.

**Comment by ThaliaBarrera** [comment]
@bishalbera sure, you can work on this. Since you start it before the deadline we can count it in, no worries. I'm assigning it to you. 

About working on these issues after the hackathon, yes, that's definitely an option. I'll contact you via Slack if you're okay and we can discuss.

**Comment by bishalbera** [comment]
@ThaliaBarrera Ya I'm absolutely ok with that. Ty also plz check the PR I've mentioned some issues there

**Comment by bishalbera** [comment]
@ThaliaBarrera do we have to submit all the PR before the hackathon ends?

**Comment by bishalbera** [comment]
@ThaliaBarrera I'm facing a problem. when I'm directly handling the API request, I'm getting all the data, including population, income, and other details. However, after syncing the API with BigQuery, I'm getting null values for all the fields. I've attached the necessary pictures. Also, I'm running short on time.

![image](https://github.com/airbytehq/quickstarts/assets/123734227/c31c3f68-5889-4878-8dac-1b5c39416546)

![image](https://github.com/airbytehq/quickstarts/assets/123734227/4638ff75-ffdd-4f4e-86e0-7658102c1e6f)

**Comment by ThaliaBarrera** [comment]
I have no idea what's going on @bishalbera! I need to try to reproduce the issue myself, but we're running out of time with the hackathon. So perhaps we cannot count this in 😞 

We can continue working on this quickstart but outside of the hackathon, I'll DM you.

**Comment by bishalbera** [comment]
@ThaliaBarrera that's ok . I'm thinking if take another issue and raise the pr after the hackathon ends will that count???

**Comment by ThaliaBarrera** [comment]
Yes, sure, we can count it in! Just let me know which one you'd like to take @bishalbera

**Comment by bishalbera** [comment]
@ThaliaBarrera Thats awesome then I would like to work on [this](https://github.com/airbytehq/quickstarts/issues/24) one. Im thinking to use shopify api for that issue but the thing is my trial period is over so the api key will not work. Is it possible to give me the credentials for that or should I just use another email for it.
